src/api-models/fba-inventory-api-model/base.ts

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable */
 /**
  * Selling Partner API for FBA Inventory
- * The Selling Partner API for FBA Inventory lets you programmatically retrieve information about inventory in Amazon\'s fulfillment network.
+ * The Selling Partner API for FBA Inventory lets you programmatically retrieve information about inventory in Amazon\'s fulfillment network. Today this API is available only in the North America region. In 2021 we plan to release this API in the Europe and Far East regions.
```

Comment:
Interesting that there are other changes here, unrelated to ENUM and unions. That means our auto updater is not working?

Reply:
I've removed unrelated changes. Thanks.

Reply:
> That means our auto updater is not working?

I re-generated models. Therefore, there were unrelated changes on this PR.